### ISSUE TITLE
Sanitize SVG properly

### ISSUE THREAD

**Issue by Frenzie** [FreshRSS/FreshRSS#2139]
Cf. #2135, #2106.

Example feed with at least some exposed SVG:

https://fransdejonge.com/category/media/drawing/feed/

**Comment by Alkarex** [comment]
The SVG seems to be directly in your RSS feed, without having to fetch full content

**Comment by Frenzie** [comment]
You're right. The Fx preview only shows the summary but that's otherwise unrelated to the feed itself, which I didn't realize. I've gone back and forth on the matter of full feeds over the past decade, and I forgot about settling on this:

```
<p>The post <a href="https://fransdejonge.com/2017/08/the-ghost-of-student-counseling/">The Ghost of Student Counseling</a> was written by Frans at <a href="https://fransdejonge.com">The One with the Thoughts of Frans</a>.</p>
```

There are these odd search engine bait sites that used to slurp up all the feeds they could find. Or perhaps they still exist.

**Comment by math-GH** [comment]
#2135, #2106 are fixed/closed.
What else is to do here?

**Comment by Frenzie** [comment]
Stripping them out is a workaround; we'd like to actually display them.

**Comment by math-GH** [comment]
for me it works fine already. The SVG file is shown
![grafik](https://user-images.githubusercontent.com/1645099/129441551-68d4d917-dd52-4e4f-8629-9a8adac0aca8.png)

**Comment by Frenzie** [comment]
Was that an inline SVG or simply an SVG in an IMG?

**Comment by math-GH** [comment]
SVG in an img

![grafik](https://user-images.githubusercontent.com/1645099/129442206-24f63c6e-985b-4867-a798-68925be0a7aa.png)

**Comment by math-GH** [comment]
Now I understand. The issue is with "plain" `<svg>`

**Comment by math-GH** [comment]
Do we have an example feed that has `<svg>` included?

**Comment by Frenzie** [comment]
My feed above has two. That's just not one of them. ;-)

**Comment by mathContao** [comment]
this blog post?
https://fransdejonge.com/2016/04/trois-champignons/
![grafik](https://user-images.githubusercontent.com/1645099/129452893-d1916c2f-5253-4556-94cf-c9455577407b.png)

On your page: If I delete the `<img>` line that I see that the svg does not load.

**Comment by thefranke** [comment]
Hello, I want to chime in on regular use cases for SVG in feeds. I have a personal blog that pre-renders Mathjax into SVG, as most non-web feed-readers do not have built-in support for Mathjax and will show raw Latex math formulas instead of a nicely rendered formula.

Alternatively I could also switch to MathML, but SVG seems like a legitimate way to render them inside a feed.

**Comment by Inverle** [comment]
ref: https://www.tobias-franke.eu/rss/index.xml
